git-bootstrap.yaml

```diff
@@ -0,0 +1,159 @@
+             no-mdc2 \
+             no-rc5 \
+             no-ec2m \
+             no-sm2 \
+             no-sm4 \
+             no-ssl3 \
+             no-seed \
+             no-weak-ssl-ciphers \
+             no-docs \
+             -Wa,--noexecstack
+          perl configdata.pm --dump
+          make -j$(nproc)
+          make install
+
+  - working-directory: curl
+    pipeline:
+      - uses: fetch
+        with:
+          uri: https://curl.se/download/curl-${{vars.curl-version}}.tar.xz
+          expected-sha256: 6fea2aac6a4610fbd0400afb0bcddbe7258a64c63f1f68e5855ebc0c659710cd
+      - uses: autoconf/configure
+        with:
+          opts: --with-openssl --disable-docs --disable-manual --disable-shared
+      - runs: |
+          make -C lib/ -j$(nproc)
+          make -C src/ -j$(nproc)
+          make -C include/ -j$(nproc)
+          make -C lib/ install
+          make -C src/ install
+          make -C include/ install
+
+  - working-directory: git
+    pipeline:
+      - uses: fetch
+        with:
+          uri: https://www.kernel.org/pub/software/scm/git/git-${{package.version}}.tar.xz
+          expected-sha256: 51bfe87eb1c02fed1484051875365eeab229831d30d0cec5d89a14f9e40e9adb
+      - runs: |
+          cat >> config.mak <<-EOF
+          NO_GETTEXT=YesPlease
+          NO_SVN_TESTS=YesPlease
+          NO_REGEX=YesPlease
+          NO_SYS_POLL_H=1
+          ICONV_OMITS_BOM=Yes
+          INSTALL_SYMLINKS=1
+          NO_PERL=YesPlease
+          NO_TCLTK=YesPlease
+          NO_EXPAT=YesPlease
+          EOF
+      - runs: |
+          make prefix=/usr \
+            CFLAGS="-O2 -Wall" \
+            DESTDIR="${{targets.destdir}}" \
+            INSTALLDIRS=vendor \
+            CURL_LDFLAGS="-l:libcurl.a -lssl -lcrypto" \
+            install -j$(nproc)
+      - runs: |
+          mkdir -p "${{targets.destdir}}"/var/git
+
+update:
+  enabled: true
+  release-monitor:
+    identifier: 5350
+
+test:
+  pipeline:
+    - name: Verify git installation
+      runs: |
+        git --version || exit 1
+    - name: Basic git operations
+      runs: |
+        git config --global user.name "Wolfi"
+        git config --global user.email "wolf@wolfi.dev"
+        mkdir test_repo && cd test_repo
+        git init
+        touch README.md
+        git add README.md
+        git commit -m "Initial commit"
+        git status | grep 'nothing to commit, working tree clean' || exit 1
+    - name: Clone repository
+      runs: |
+        git clone --depth 1 https://github.com/wolfi-dev/os.git test_clone
+        [ -d "test_clone/.git" ] || exit 1
+    - name: Fetch repository updates
+      runs: |
+        cd test_clone
+        git fetch || exit 1
+    - name: Check git configuration
+      runs: |
+        git config --list | grep 'user.name=Wolfi' || exit 1config --list | grep 'user.name=Test User' || exit 1
+        git config --list | grep 'user.email=wolf@wolfi.dev' || exit 1ist | grep 'user.email=test@example.com' || exit 1
```

Comment:
`exit 1ist` and `exit 1config --list` ?

I see that you copied that, but it looks just like nonsense to me.

And , i see that @imjasonh fixed that test yesterday e1de9032301294948e48cd1897fe8b23778d83a4

So, please update to grab that change.